subprojects/core/src/main/resources/org/gradle/initialization/removed-types.txt

```diff
@@ -23,3 +23,5 @@ org.gradle.api.artifacts.transform.ArtifactTransform
 org.gradle.api.artifacts.transform.ArtifactTransformException
 org.gradle.api.artifacts.transform.VariantTransform
 org.gradle.api.tasks.incremental.IncrementalTaskInputs
+org.gradle.workers.WorkerConfiguration
```

Comment:
Whats the reasoning for adding these removed classes here? Looking at `DefaultLegacyTypesSupport`, these should be for types which "baked into the bytecode generated by the Groovy compiler."